docs/orchestration/server/deploy-local.md

```diff
@@ -66,6 +66,22 @@ Every time you run the `prefect server start` command [a set of alembic migratio
 applied against the database to ensure the schema is consistent. To run the migrations directly please
 see the documentation on [prefect server migrations](https://github.com/PrefectHQ/server#running-the-system).
 
+### External Postgres instance
+
+If you want Prefect Server to use an external Postgres instance, configure the database url in `~/.prefect/config.toml` on whatever machine you're running Prefect Server:
+
+```
+[server.database]
+connection_url = "postgres://<username>:<password>@hostname:<port>/<dbname>"
+```
+
+This connection url will be passed directly to Hasura. For more information and troubleshooting, see the [docs](https://hasura.io/docs/latest/graphql/core/deployment/deployment-guides/docker.html#database-url).
+
+After setting the connection url, run `prefect server start --external-postgres`.
```

Comment:
Is it a pain to make `--external-postgres` take an optional string that sets the connection url?

Reply:
(we'd still fall back on the config value)

Reply:
it is quite a pain, would a separate flag that provides the connection string be helpful?
`prefect server start --external-postgres` # config value
`prefect server start --external-postgres-url postgres://my-database-url.com` # use the input directly

Reply:
That makes sense I think.


Does this make any sense?
```
prefect server start --external-postgres --postgres-url postgres://my-database-url.com
```

or is it completely worthless in the case where you're not using external postgres?
```
prefect server start --postgres-url postgres://localhost/
```

Reply:
`postgres-url` only has meaning if we're using external postgres

seems fair to be flexible in supporting  three cases

1. using config value
`prefect-server-start --external-postgres`
2. providing a url explicitly (we'll set `external_postgres` to `True`)
`prefect server start --postgres-url postgres://my-database-url.com`
3. providing a url explicitly + redundant cli flag (makes no sense but I don't see why a user should hit an error when their intention is clear)
`prefect server start --external-postgres --postgres-url postgres://my-database-url.com`